t/basic.t

```diff
@@ -12,19 +12,15 @@ diag join "\t", qw{Mod Lib};
 diag join "\t", $Alien::libdeflate::VERSION, Alien::libdeflate->version;
 
 SKIP: {
-  skip "system install", 9 if Alien::libdeflate->install_type eq 'system';
+  skip "system install", 6 if Alien::libdeflate->install_type eq 'system';
 
-  run_ok(['gzip', '-h'])
+  run_ok(['libdeflate-gzip', '-h'])
     ->success
-    ->out_like(qr/^usage:\s+gzip\s/mi);
+    ->out_like(qr/^usage:\s+libdeflate-gzip(\.exe)?\s/mi);
 
-  run_ok(['gunzip', '-h'])
+  run_ok(['libdeflate-gunzip', '-h'])
     ->success
-    ->out_like(qr/^usage:\s+gunzip\s/mi);
-
-  run_ok(['checksum', '-h'])
-    ->success
-    ->out_like(qr/^usage:\s+checksum\s/mi);
+    ->out_like(qr/^usage:\s+libdeflate-gunzip(\.exe)?\s/mi);
```

Comment:
Is this correct? Should it be `libdeflate-checksum`?

Reply:
I’ve removed `checksum` completely in one of the commits.

Reply:
It’s a test program: https://github.com/ebiggers/libdeflate/blob/v1.10/Makefile#L264